### PR TITLE
Needed params from chain status

### PR DESCRIPTION
**Pull request by VoR0220**
Needed the latest block height, so I plopped that in there.

### CODE REVIEW

File: client/cmd/transaction.go [hunk 1]
```diff
@@ -76,13 +76,13 @@ func buildTransactionCommand() {
...
 		Run:   func(cmd *cobra.Command, args []string) {
-			// transaction.Call(clientDo)
+			transaction.Call(clientDo)
 		},
 		PreRun: assertParameters,
 	}
 	callCmd.Flags().StringVarP(&clientDo.AmtFlag, "amt", "a", "", "specify an amount")
 	callCmd.Flags().StringVarP(&clientDo.ToFlag, "to", "t", "", "specify an address to send to")
-	callCmd.Flags().StringVarP(&clientDo.DataFlag, "data", "d", "", "specify some data")
+	callCmd.Flags().StringVarP(&clientDo.DataFlag, "data", "", "", "specify some data")
```
Comment (silasdavis) [inline client/cmd/transaction.go]: If you're killing the shorthand name, shouldn't you be using `StringVar`. Looking at the source I see this has the same effect (https://github.com/ogier/pflag/blob/master/string.go#L34) but you intention would be clearer and you would not be exposed to a change of implementation giving you undefined behaviour if you used `StringVar`

**Comment by silasdavis** [comment]
I'm not entirely sure what this is for, so I will leave @benjaminbollen to do final signoff

File: client/cmd/transaction.go (hunk 1 above)
Comment (benjaminbollen) [inline client/cmd/transaction.go]: P for cobra is persistent, and the need to remove d compared to old mint-client is due to introduction of -d --debug on top level command

Comment (silasdavis) [inline client/cmd/transaction.go]: OK, just saying `StringVar` is more appropriate here than `StringVarP`

**Comment by benjaminbollen** [comment]
will track in #264

**Pull request closed without merging**